x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -95,23 +94,21 @@ const installPreBuiltTemplates = async (paths: string[], callCluster: CallESAsCu
       templateAPIPath = '_index_template';
     }
 
-    const callClusterParams: {
+    const esClientParams: {
       method: string;
       path: string;
-      ignore: number[];
       body: any;
     } = {
       method: 'PUT',
       path: `/${templateAPIPath}/${templateName}`,
-      ignore: [404],
       body: content,
     };
     // This uses the catch-all endpoint 'transport.request' because there is no
     // convenience endpoint using the new _index_template API yet.
     // The existing convenience endpoint `indices.putTemplate` only sends to _template,
     // which does not support v2 templates.
     // See src/core/server/elasticsearch/api_types.ts for available endpoints.
```

Comment:
The new client should have a specific function to install v2 index templates: 

https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#_indices_putindextemplate

The fallback to `transport.request` shouldn't be necessary any more.